tests/MockTest/PaymentTest.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace Adyen\MockTest;
+
+use Monolog\Handler\StreamHandler;
+
+class PaymentTest extends TestCaseMock
+{
+    /**
+     * @param $jsonFile Json file location
+     * @param $httpStatus expected http status code
+     *
+     * @dataProvider successAuthoriseProvider
+     */
+    public function testAuthoriseSuccess($jsonFile, $httpStatus)
+    {
+        // create client
+        $client = $this->createMockClient($jsonFile, $httpStatus);
+
+        // Set Logger on level INFO and capture on memory
+        $stream = fopen('php://memory','r+');
+        $client->getLogger()->pushHandler(new StreamHandler($stream, \Monolog\Logger::INFO));
+
+        // initialize service
+        $service = new \Adyen\Service\Payment($client);
+
+        $json = '{
+              "card": {
+                "number": "4111111111111111",
+                "expiryMonth": "08",
+                "expiryYear": "2018",
+                "cvc": "737",
+                "holderName": "John Smith"
+              },
+              "amount": {
+                "value": 1500,
+                "currency": "EUR"
+              },
+              "reference": "payment-test",
+              "merchantAccount": "YourMerchantReference"
+            }';
+
+        $params = json_decode($json, true);
+
+        $result = $service->authorise($params);
+
+        // Rewind the stream and save the INFO logs in a variable
+        rewind($stream);
+        $infoLogs = stream_get_contents($stream);
+        fclose($stream);
+
+        $this->assertArrayHasKey('resultCode', $result);
+        $this->assertEquals('Authorised', $result['resultCode']);
+        $this->assertContains("JSON Request to Adyen:{\"card\":{\"number\"", $infoLogs);
+        $this->assertNotContains("4111111111111111", $infoLogs);
```

Comment:
to fully cover https://github.com/Adyen/adyen-php-api-library/blob/develop/src/Adyen/HttpClient/CurlClient.php#L249 we better cover cvc and cse encrypted blob